docusaurus/blog/authors.yml

```diff
@@ -1,17 +1,23 @@
-endi:
-  name: Endilie Yacop Sucipto
-  title: Maintainer of Docusaurus
-  url: https://github.com/endiliey
-  image_url: https://github.com/endiliey.png
+andrew_martinez:
+  name: Andrew Martinez
+  title: OpenZiti Maintainer
+  url: https://hashnode.com/@andrewpmartinez
+  image_url: /img/profile-andrew-martinez.webp
 
-yangshun:
-  name: Yangshun Tay
-  title: Front End Engineer @ Facebook
-  url: https://github.com/yangshun
-  image_url: https://github.com/yangshun.png
+paul_lorenz:
```

Comment:
would there be value in using the github handle? feels that way but maybe not?

Reply:
Done